src/components/forms/SemanticLink/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+
+
```

Comment:
We should leave a prominent comment in this file to remind people that, although this is called the SemanticLink component, we are not yet displaying as a SemanticLink and only showing the value as a read only text field.